src/Windowing/Silk.NET.GLFW/Enums/ClientApi.cs

```diff
@@ -0,0 +1,31 @@
+//
+// This file is part of Silk.NET.
+//
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+namespace Silk.NET.GLFW
+{
+    /// <summary>
+    /// The context client APIs.
+    /// </summary>
+    /// <seealso cref="IGLFW.WindowHint(WindowHintClientApi,ClientApi)"/>
+    public enum ClientApi
+    {
+        /// <summary>
+        /// No context API is created.
+        /// </summary>
+        NoApi = 0,
+
+        /// <summary>
+        /// OpenGL context is created.
+        /// </summary>
+        OpenGlApi = 0x00030001,
```

Comment:
If we're being pernickety about capitalisation, might as well fix it here too.

Reply:
Should it become OpenGLAPI or OpenGLApi?

Reply:
How about just OpenGL?